test/es6/ES6PromiseAsync.baseline

```diff
@@ -135,6 +153,25 @@ Test #56 - Success finally handler called for rejected promise without value
 Test #63 - val is 0(Expect 0)
 Test #66 - Success - []
 Test #68 - Failed - {}
+Test #76 - Catch handler #1 called with err = TypeError: Function expected
+Test #76 - Catch handler #1 called with err.message = Function expected
+Test #77 - Catch handler #1 called with err = TypeError: Function expected
+Test #77 - Catch handler #1 called with err.message = Function expected
+Test #78 - Catch handler #1 called with err = TypeError: Object expected
+Test #78 - Catch handler #1 called with err.message = Object expected
+Test #79 - Catch handler #1 called with err = TypeError: Function expected
+Test #79 - Catch handler #1 called with err.message = Function expected
+Test #80 - Catch handler #1 called with err = TypeError: failure inside iterator
+Test #80 - Catch handler #1 called with err.message = failure inside iterator
+Test #81 - Error handler #1 called with err = TypeError: failure inside iterator
+Test #81 - Error handler #1 called with err.message = failure inside iterator
+Test #88 - Error handler #1 called with err = AggregateError: Promise.any all promises rejected﻾﻾﻾﻾﻾﻾﻾̃ﾂ￿

```

Comment:
Sorry. I'm not sure what happend.

Reply:
Looks like a text encoding issue? Whatever it was it's caused the CI to fail.

Reply:
My bad. resolved!